### ISSUE TITLE
[REQ] Change dragging function in 'Files' tab 

### ISSUE THREAD

**Issue by zeromido** [drizt/torrent-file-editor#8]
In 'Files' tab it would be better if dragging a file causes it to move it up or down not select, and to select files dragging the mouse pointer to create a selection around the outside of all the items (like windows explorer) and you can do that by leaving blank spaces between rows.
That's the dragging selection I meant
http://i.imgur.com/PYVHL9P.png
Also, when dragging files to move them up or down it would be better if they move with their icons and names (like windows explorer)
In other words:
See how windows explorer selects and moves files that's what I'm asking you to do :)

**Comment by zeromido** [comment]
@drizt 
I removed the move up and down related text because I just realized that files order won't be saved in the .torrent files. So, it would be unnecessary. You may even think about removing the up and down buttons!

**Comment by drizt** [comment]
.torrent file holds files order. I don't understand why you said 'won't be saved'.

**Comment by zeromido** [comment]
Really but when I change the order the app doesn't tell me about any changes.

**Comment by drizt** [comment]
np.

Also you need to remember that when you change the files order hash of torrent file will be changed.

**Comment by zeromido** [comment]
OK

**Comment by drizt** [comment]
Moving files feature on Files tab is very basic. It should be improved.

**Comment by drizt** [comment]
I will add such feature. But not in the nearest version.
